lib/models/device.coffee

```diff
@@ -101,13 +101,26 @@ getDeviceModel = (deps, opts) ->
 
 	# Internal method for uuid/id disambiguation
 	# Note that this throws an exception for missing uuids, but not missing ids
+	# method for staged-releases get-device-id.sh
 	getId = (uuidOrId) ->
 		Promise.try ->
 			if isId(uuidOrId)
 				return uuidOrId
 			else
 				exports.get(uuidOrId, $select: 'id').get('id')
 
+	###*
+	# @summary Get device id for a specific device on a resin.io application
+	# @param {String} uuid - uuid of specific device
+	# @memberof resin.models.device
+	# @returns {Number} DeviceId
+	# @example
+	# resin.models.device.getDeviceId(device.uuid)
+	###
+	exports.getDeviceId = (uuid, callback) ->
```

Comment:
Why do we need this as a public method? I'm not sure it's something users should be calling. It's part of staged releases, but I think just to use internally, not something users should need to care about.

Reply:
Given that we were trying to not use device ids in any of our methods, I agree to not add this as part of the public methods.